system/greenhouse-ccloud/templates/kube-monitoring-pluginpreset.yaml

```diff
@@ -52,19 +72,32 @@ spec:
         - __name__
         targetLabel: cluster
   {{- end }}
-  {{- end }}
-  {{- range $cluster, $thanos := .Values.kubeMonitoring.computeClusters }}
+  {{- range $cluster, $thanos := .Values.kubeMonitoring.obsClusters }}
   - clusterName: {{ $cluster }}
     overrides:
     - name: global.commonLabels
       value:
-        service: kvm
+        service: observability
     - name: kubeMonitoring.prometheus.prometheusSpec.externalLabels
       value:
         cluster: '{{ "{{ .Values.global.greenhouse.clusterName }}" }}'
-        cluster_type: compute
+        cluster_type: observability
```

Comment:
Why was this on `kvm` and `compute`?

Reply:
It's still there. Line 59 and 63. Git is doing some strange diff here.

Reply:
Ah my bad 👍